decred/examples/create_testnet_wallet.py

```diff
@@ -9,11 +9,11 @@
 from getpass import getpass
 
 from decred import config
+from decred.util.helpers import mkdir
+from decred.wallet.wallet import Wallet
 
 # Set the configuration for testnet before loading TD modules.
 config.load("testnet")
-from decred.util.helpers import mkdir
-from decred.wallet.wallet import Wallet
```

Comment:
The breaks the script. I have a solution in mind so that this won't be needed, but I haven't worked out the details yet. For now, it needs to stay as is.

Reply:
> I think teknico is on it, and he'll want to configure things, I imagine.

I am and I want to. While black is almost non-configurable, isort has way too many options and its default behaviour conflicts with black, as seen above. I'm working out the isort config changes needed to make it coexist peacefully with black...

> This breaks the script.

...and avoid problems like this one. (And no, we're not the first project with this problem. :slightly_smiling_face: )

> I have a solution in mind so that this won't be needed, but I haven't worked out the details yet.

Minimizing import-time side effects is desirable indeed.